lib/guardian.rb

```diff
@@ -398,7 +401,12 @@ def can_invite_to?(object, groups = nil)
     if object.is_a?(Topic)
       if object.private_message?
         return true if is_admin?
-        return false unless SiteSetting.enable_personal_messages?
+
+        # TODO (martin) Remove enable_personal_messages here once plugins have been changed.
+        if !@user.in_any_groups?(SiteSetting.group_setting_map(:personal_message_enabled_groups)) ||
```

Comment:
There are many such places in the backend where I need to leave `enable_personal_messages` in place, if the condition was removed then plugin tests would fail since they are not setting up user auto groups based on trust level correctly. Once I fix the plugins and update them to remove usages of `enable_personal_messages` and `min_trust_to_send_messages` then we will be good to remove this from core.